packages/transaction-log/src/components/common/app.main.component.tsx

```diff
@@ -0,0 +1,339 @@
+            filters.channelID = channel
+          }
+
+          if (reruns !== 'NoFilter') {
+            if (reruns === 'Yes') {
+              filters.childIDs = JSON.stringify({$exists: true, $ne: []})
+            } else if (reruns === 'No') {
+              filters.childIDs = JSON.stringify({$eq: []})
+            }
+          }
+
+          if (startDate && endDate) {
+            filters['request.timestamp'] = JSON.stringify({
+              $gte: startDate.toISOString(),
+              $lte: endDate.toISOString()
+            })
+          }
+
+          if (host) {
+            filters['request.host'] = host
+          }
+
+          if (port) {
+            filters['request.port'] = port
+          }
+
+          if (path) {
+            filters['request.path'] = path
+          }
+
+          if (param) {
+            filters['request.querystring'] = param
+          }
+
+          if (client !== 'NoFilter') {
+            filters.clientID = client
+          }
+
+          if (method !== 'NoFilter') {
+            filters['request.method'] = method
+          }
+        }
+
+        const fetchParams: {[key: string]: any} = {
+          filters: JSON.stringify(filters)
+        }
+        if (!timestampFilter) {
+          fetchParams.filterLimit = limit
+          fetchParams.filterPage = 0
+        }
+
+        const transactions = await getTransactions(fetchParams);
+
+        const transactionsWithChannelDetails = await Promise.all(
+          transactions.map(async transaction => {
+            const channelName = await fetchChannelDetails(transaction.channelID)
+            const clientName = await fetchClientDetails(transaction.clientID)
+            return {...transaction, channelName, clientName}
+          })
+        )
+
+        setTransactions(transactionsWithChannelDetails)
+      } catch (error) {
+        console.error('Error fetching logs:', error)
+      }
+    },
+    [
+      status,
+      statusCode,
+      channel,
+      startDate,
+      endDate,
+      limit,
+      reruns,
+      host,
+      port,
+      path,
+      param,
+      client,
+      method
+    ]
+  )
+
+  const fetchAvailableChannels = useCallback(async () => {
+    try {
+      const channels = await getChannels()
+      setChannels(channels)
+    } catch (error) {
+      console.error('Error fetching channels:', error)
+    }
+  }, [])
+
+  const fetchAvailableClients = useCallback(async () => {
+    try {
+      const clients = await getClients()
+      setClients(clients)
+    } catch (error) {}
+  }, [])
+
+  // useEffect(() => {
+  //   const intervalId = setInterval(async () => {
+  //     const timestamp = new Date().toISOString()
+  //     await fetchTransactionLogs('2024-08-14T15:50:25+03:00') // Auto-polling with timestamp filter, without filterLimit and filterPage
+  //   }, 5000)
+
+  //   return () => clearInterval(intervalId) // Cleanup interval on component unmount
+  // }, [fetchTransactionLogs])
```

Comment:
Remove unused codes

Reply:
Seems we still have the commented code. Is it useful?

Reply:
I have removed them and do not exist in my local machine. @drizzentic